test/README.external

```diff
@@ -67,3 +71,30 @@ within the OpenSSL source code.
 The community is encouraged to contribute patches which reduce the number of
 suppressions that are currently present.
 
+
+Python PYCA/Cryptography test suite
+===================================
+
+This python test suite runs cryptographic tests with a local OpenSSL build as
+the implementation.
+
+First checkout the PYCA/Cryptography module into ./pyca-cryptography using:
+
+$ git submodule update --init
+
+Then configure/build OpenSSL compatible with the python module:
+
+$ ./config shared enable-external-tests
+$ make
+
+The tests will run in a python virtual environment which requires virtualenv
+to be installed.
+
+$ HARNESS_VERBOSE=1 TESTS="test_external" make test
```

Comment:
Is it too much to ask to not assume that every prompts in the world is Bourne-shell one? I mean the way environment variables are set. There is shell-neutral way after all, `env VAR=VALUE ...`. In the context one can also move them to the right of command line... This is a side note, i.e. no specific action is expected.